### DISCUSSION TITLE
t is not define error when i use component in template in nuxt 3 rc 4

### DISCUSSION THREAD

**Discussion by shahensaifullah** [epicmaxco/vuestic-ui#2044] [General]
![image](https://user-images.githubusercontent.com/44544213/177416532-2825f3ae-93c9-418c-b077-100de69fef32.png)
index.vue
```
<template>
  <div>
    <va-button> Button </va-button>
  </div>
</template>

<script setup>

</script>
```
nuxt.config.ts
```
import { defineNuxtConfig } from 'nuxt'

// https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
    buildModules: [
        'nuxt-windicss',
    ],
    modules: ['@vuestic/nuxt'],
})

```
when i use component in template, it shows this error. but if I don't use any component in template, it doesn't show any errors.

**Comment by m0ksem** [comment]
@shahriar350, hey! Must be fixed in 1.4.6. 👌 Simply reinstall @vuestic/nuxt.
